uptane-standard.md

```diff
@@ -864,7 +864,7 @@ In order to perform full verification, an ECU SHALL perform the following steps:
     1. Locate and download a Targets metadata file from the Image repository that contains an image with exactly the same filename listed in the Director metadata, following the procedure in {{resolve_delegations}}.
     2. Check that the Targets metadata from the Image repository matches the Targets metadata from the Director repository:
         1. Check that the non-custom metadata (i.e., length and hashes) of the unencrypted or encrypted image are the same in both sets of metadata. Note: the Primary is responsible for validating encrypted images and associated metadata. The target ECU (Primary or Secondary) is responsible for validating the unencrypted image and associated metadata.
-        2. Check that all "MUST match" custom metadata (e.g., hardware identifier and release counter) are the same in both sets of metadata.
+        2. Check that all MUST match custom metadata (e.g., hardware identifier and release counter) are the same in both sets of metadata.
```

Comment:
Looks like we use this phrase "MUST match" in a few places. Is it okay to leave that as is? Otherwise I feel like we need some better way to name it.

Reply:
I looked at it and was going to change it, but it seemed to me the meaning was a bit different. And there are no commas involved to make it messy, so I was going to leave it be. Let me know if you feel different.

Reply:
I think it reads slightly better with the quotes here, but I'm fine either way. I think this is mergeable now in either case.